src/Admin/AbstractAdmin.php

```diff
@@ -2481,21 +2485,23 @@ private function buildTabMenu(string $action, ?AdminInterface $childAdmin = null
 
         $this->loaded['tab_menu'] = true;
 
-        $this->menu = $this->getMenuFactory()->createItem('root');
-        $this->menu->setChildrenAttribute('class', 'nav navbar-nav');
-        $this->menu->setExtra('translation_domain', $this->getTranslationDomain());
+        $menu = $this->getMenuFactory()->createItem('root');
+        $menu->setChildrenAttribute('class', 'nav navbar-nav');
+        $menu->setExtra('translation_domain', $this->getTranslationDomain());
 
         // Prevents BC break with KnpMenuBundle v1.x
-        if (method_exists($this->menu, 'setCurrentUri')) {
-            $this->menu->setCurrentUri($this->getRequest()->getBaseUrl().$this->getRequest()->getPathInfo());
+        if (method_exists($menu, 'setCurrentUri')) {
```

Comment:
maybe we can remove this method exists? we dont support knp menu bundle 1, do we?